pymc/tests/test_missing.py

```diff
@@ -98,7 +98,7 @@ def test_interval_missing_observations():
         assert "theta1_observed_interval__" in model.named_vars
         assert "theta1_missing_interval__" in model.named_vars
         assert isinstance(
-            model.rvs_to_values[model.named_vars["theta1_observed"]].tag.transform, Interval
+            model.rvs_to_values[model.named_vars["theta1_observed"]].tag.transform, interval
```

Comment:
`interval` is a type? That's unexpected from a lowercase name =/

Reply:
It already is the case, no ?

https://github.com/pymc-devs/pymc3/blob/8ae296bdf3ee58850824c9997293803d2df958ec/pymc/distributions/transforms.py#L224

It's just that the actual name of the class went from `Interval` to `IntervalTransform`